hack/deploy_loki_using_observatorium_yaml.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+LOKI_PROJECT_NAME=observatorium-logs-test
+LOKI_S3_SECRET_NAME=loki-objectstorage-secret
+AWS_CREDENTIALS_FILE_NAME=~/.aws/credentials
+
+LOKI_INGESTER_REPLICAS=${LOKI_INGESTER_REPLICAS:-"3"}
+
+get_aws_credentials() {
+  # get AWS credentials from local user account file
+  AWS_S3_USER_ACCESS_KEY=$(grep -m 1 aws_access_key_id < $AWS_CREDENTIALS_FILE_NAME | awk '{print $3}')
+  AWS_S3_USER_SECRET_KEY=$(grep -m 1 aws_secret_access_key < $AWS_CREDENTIALS_FILE_NAME | awk '{print $3}')
+
+  # Get AWS credentials from environment variables
+  AWS_S3_END_POINT="${AWS_S3_END_POINT:-https://s3.us-east-1.amazonaws.com}"
+  AWS_S3_REGION="${AWS_S3_REGION:-us-east-1}"
+  AWS_S3_LOKI_BUCKET_NAME="${AWS_S3_LOKI_BUCKET_NAME:-loki-bucket}"
+  AWS_S3_ACCESS_KEY="${AWS_S3_ACCESS_KEY:-$AWS_S3_USER_ACCESS_KEY}"
+  AWS_S3_SECRET_KEY="${AWS_S3_SECRET_KEY:-$AWS_S3_USER_SECRET_KEY}"
+
+  if [ -z "$AWS_S3_ACCESS_KEY" ]; then
+    echo "==> ERROR: Provide AWS access_key using AWS_S3_ACCESS_KEY env.variable or using $AWS_CREDENTIALS_FILE_NAME file"
+    exit
+  fi
+
+  if [ -z "$AWS_S3_SECRET_KEY" ]; then
+    echo "==> ERROR: Provide AWS secret_key using AWS_S3_SECRET_KEY env.variable or using $AWS_CREDENTIALS_FILE_NAME file"
+    exit
+  fi
+
+  echo "
+Using AWS S3 configuration for loki bucket::
+-==-=-=-=-=-=-
+AWS S3 end-point: $AWS_S3_END_POINT
+AWS S3 region: $AWS_S3_REGION
+
+bucket name: $AWS_S3_LOKI_BUCKET_NAME
+access_key: $AWS_S3_ACCESS_KEY
+secret_key: $AWS_S3_SECRET_KEY
+
+(Note: Use environment variables as needed to change defaults)
+"
+
+
+}
+
+check_oc_works() {
+  OC_WORKS=$(oc get projects > /dev/null 2>&1 ; echo $?)
+  if ! [ "$OC_WORKS" == "0" ]; then
+    echo "==> ERROR: oc command not working,  make sure you are logged into OCP cluster and that oc command works"
+    exit
+  else
+    echo "OK, OCP cluster working"
+  fi
+}
+
+delete_project_if_exists() {
+  PROJECT=$(oc get project | grep "$1")
+  if [ -n "$PROJECT" ]; then
+    echo "Deleting $1 project/namespace"
+    oc delete project "$1"
+    while : ; do
+      PROJECT=$(oc get project | grep "$1")
+      if [ -z "$PROJECT" ]; then break; fi
+      sleep 1
+    done
+  fi
+}
+
+create_project() {
+  oc new-project "$1" > /dev/null 2>&1
+  echo "Done"
+}
+
+get_observatorium_template() {
+  rm -f observatorium-logs-template.yaml
+  wget -nv https://raw.githubusercontent.com/rhobs/configuration/main/resources/services/observatorium-logs-template.yaml
```

Comment:
I suggest to default to `curl -L -O` because `wget` is usually not per default available on all platforms. In addition we can use a tmp directory for downloaded files.

Reply:
wget is common ... will stay with that for now
changed location to temp (also for the secret) <-- DONE